website/project/views/node.py

```diff
@@ -735,6 +735,13 @@ def _view_project(node, auth, primary=False,
             'institutions': get_affiliated_institutions(node) if node else [],
             'has_draft_registrations': node.has_active_draft_registrations,
             'is_preprint': node.is_preprint,
+            'has_moderated_preprint': node.has_moderated_preprint,
+            'preprint_state': node.linked_preprint.reviews_state if node.has_moderated_preprint else '',
+            'preprint_word': node.linked_preprint.provider.preprint_word if node.is_preprint else '',
+            'preprint_provider': {
+                'name': node.linked_preprint.provider.name,
+                'workflow': node.linked_preprint.provider.reviews_workflow
```

Comment:
Since properties aren't cached, this is basically the same as before (doing `get_queryset()[0]` over and over). It'd be nicer to get the preprint higher up in this function, then use the variable.